make/migrations/postgresql/0011_1.10.0_schema.up.sql

```diff
@@ -0,0 +1,35 @@
+/*Table for keeping the plug scanner registration*/
+CREATE TABLE scanner_registration
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    uid VARCHAR(64) UNIQUE NOT NULL,
+    url VARCHAR(256) UNIQUE NOT NULL,
+    name VARCHAR(128) UNIQUE NOT NULL,
+    description VARCHAR(1024) NULL,
+    auth VARCHAR(16) NOT NULL,
+    access_cred VARCHAR(512) NULL,
+    adapter VARCHAR(128) NOT NULL,
+    vendor VARCHAR(128) NOT NULL,
+    version VARCHAR(32) NOT NULL,
+    disabled BOOLEAN NOT NULL DEFAULT FALSE,
+    is_default BOOLEAN NOT NULL DEFAULT FALSE,
+    skip_cert_verify BOOLEAN NOT NULL DEFAULT TRUE,
+    create_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    update_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+/*Table for keeping the scanner report. The report details are stored as JSONB*/
+CREATE TABLE scanner_report
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    digest VARCHAR(256) NOT NULL,
+    endpoint_id VARCHAR(64) NOT NULL,
+    vendor VARCHAR(32) NOT NULL,
+    job_id VARCHAR(32),
+    status VARCHAR(16) NOT NULL,
+    status_code INTEGER DEFAULT 0,
+    report JSONB,
```

Comment:
What's the reason to choose JSONB over `text` or `JSON` do we want to index the value in this column?

Reply:
In case we have the query requirement in future

Reply:
JSON